Packs/RSANetWitness_v11_1/Scripts/RSAGetRawLog/RSAGetRawLog.py

```diff
@@ -0,0 +1,157 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import json
+
+
+''' STANDALONE FUNCTION '''
+
+
+class RSAError(Exception):
+    # exception when an element is not found
+    pass
+
+
+def is_json(myjson):
+    try:
+        json_object = json.loads(myjson)
+        return json_object
+    except ValueError:
+        return None
+
+
+def merge_dict(first_dict: dict, second_dict: dict):
+    res = first_dict.copy()
+    for key, value in second_dict.items():
+        if key in res:
+            if isinstance(res[key], list):
+                if isinstance(value, list):
+                    res[key] = list(set(res[key] + value))
+                elif value not in res[key]:
+                    res[key].append(value)
+            elif value != res[key]:
+                if isinstance(value, list):
+                    res[key] = list({res[key], *value})
+                else:
+                    res[key] = [res[key], value]
+        else:
+            res[key] = value
+    return res
+
+
+def get_raw_log(event_source_id: str, concentrator_ip: str, concentrator_port: str) -> list:
+    if isCommandAvailable("netwitness-packets") is False:
+        return [{"log": "Please add RSA Netwitness Packet & Logs to see more details"}]
+
+    params = {
+        "sessions": event_source_id,
+        "concentratorIP": concentrator_ip,
+        "concentratorPort": concentrator_port,
+        "render": "application/json",
+        "renderToContext": "true"
+    }
+    res = demisto.executeCommand("netwitness-packets", params)
+
+    if type(res[0]["Contents"]) is str:
+        raise RSAError(res[0]["Contents"])
+
+    return res[0]["Contents"].get("logs")
+
+
+def get_metas_log(event_source_id: str, concentrator_ip: str, concentrator_port: str) -> list:
+    if isCommandAvailable("netwitness-packets") is False:
```

Comment:
```suggestion
    if isCommandAvailable("netwitness-query") is False:
```
Did you mean netwitness-query?

Reply:
nope, netwitness-packets is the right one